JWBestPracticeApps/FeedTableViewController/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  FeedTableViewController.swift
+//  FeedTableViewController
+//
+//  Created by Amitai Blickstein on 6/26/22.
+//
+
+import UIKit
+
+class FeedTableViewController: UITableViewController {
+    /// Instantiated with our mock/hard-coded playlist.
+    private var viewModel = FeedViewModel(withItems: Playlist.bpaManual)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        viewModel.delegate = self
+        
+        // Register the custom cell view.
+        let feedNib = UINib(nibName: viewModel.cellNibName, bundle: .main)
+        tableView.register(feedNib, forCellReuseIdentifier: viewModel.cellReuseIdentifier)
+        
+        // Various stylistic options.
+        tableView.isPagingEnabled = true
+        tableView.rowHeight = view.bounds.inset(by: view.safeAreaInsets).height
+        
+        // Must be called once to populate the table view.
+        viewModel.appendItems(fromPlaylist: Playlist.bpaManual)
```

Comment:
This line should be removed since new items will be requested until the user hit the last item in the table view.

Reply:
Done